src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -381,7 +388,8 @@ Json::Value LedgerConsensusImp::getJson (bool full)
             Json::Value ctj (Json::objectValue);
             for (auto& ct : mCloseTimes)
             {
-                ctj[beast::lexicalCastThrow <std::string> (ct.first)] = ct.second;
+                ctj[std::to_string(ct.first)] = ct.
+                second;
```

Comment:
I don't like splitting the member and the object. I'd put `ct.second` all on the second line.